config/application.rb

```diff
@@ -153,7 +153,16 @@ class Application < Rails::Application
 
     # enables in memory cache store with ttl
     # config.cache_store = TimedCachedStore.new
-    config.cache_store = :file_store, Rails.root.join("tmp", "cache")
+    rails_cache_settings = SETTINGS[:rails_cache_store]
+    if (rails_cache_settings && rails_cache_settings[:type] == 'redis')
```

Comment:
How about explicitly checking if it's a hash so a user doesn't accidentally use a string?

```suggestion
    if (rails_cache_settings.is_a?(Hash) && rails_cache_settings[:type] == 'redis')
```

I also wouldn't mind hard failing if it's something else so we don't silently fall back to files.

Reply:
I don't think it's necessary, the current code will make the app to fail if user accidentally set it to string which is actually better than silently ignoring it.